onnx_tf/common/data_type.py

```diff
@@ -55,3 +55,18 @@ def _onnx_dtype(dtype):
   else:
     raise RuntimeError("dtype should be number or str.")
   return onnx_dype
+
+
+# TODO (tjingrant) unify _onnx_dtype into any_dtype_to_onnx_dtype
+def any_dtype_to_onnx_dtype(np_dtype, tf_dtype, onnx_dtype):
```

Comment:
Is it OK that all three are positional args?

Reply:
I'm not sure what is your concern, whenever I use any_dtype_to_onnx_dtype, I always use them as keyword arguments.